README.md

```diff
@@ -141,6 +141,7 @@ See also [Rust — Production](https://www.rust-lang.org/production) organizatio
 
 * [enginesound](https://github.com/DasEtwas/enginesound) — A GUI and command line application used to procedurally generate semi-realistic engine sounds. Featuring in-depth configuration, variable sample rate and a frequency analysis window.
 * [indiscipline/zrtstr](https://github.com/indiscipline/zrtstr) — A command line utility for checking if stereo wav files are faux-stereo (i.e. have identical channels) and converting such files to mono. [<img src="https://api.travis-ci.org/indiscipline/zrtstr.svg?branch=master">](https://travis-ci.org/indiscipline/zrtstr)
+* [Lyriek](https://github.com/bartwillems/lyriek) — A multi-threaded GTK 3 application for fetching the lyrics of the current playing song. [<img src=https://github.com/bartwillems/lyriek/workflows/Rust/badge.svg?branch=master>](https://github.com/BartWillems/lyriek/actions)
```

Comment:
This is missing quotes around the `src` parameter

Reply:
fixed